### PR TITLE
fix: Forked and applied fix to BigQuery to_gcs and to_table components

### PR DESCRIPTION
**Pull request by ivanmkc**
**Description of your changes:**
Forked the `to_gcs` and `to_table components and applied a change to the datatype that switched out GCSPath for String and GCPProjectID to String.

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/component.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+metadata:
+  labels:
+    add-pod-env: 'true'
+inputs:
+  - name: query
+    description: 'The query used by Bigquery service to fetch the results.'
+    type: String
+  - name: project_id
+    description: 'The project to execute the query job.'
+    type: GCPProjectID
```
Comment (IronPan) [inline components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/component.yaml:30]: String?

File: components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/README.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+*   Output the data into a table on BigQuery.
+
+
+## Runtime arguments:
+
+
+| Argument | Description | Optional | Data type | Accepted values | Default |
+|----------|-------------|----------|-----------|-----------------|---------|
+| query | The query used by BigQuery to fetch the results. | No | String |  |  |
+| project_id | The project ID of the Google Cloud Platform (GCP) project to use to execute the query. | No | GCPProjectID |  |  |
```
Comment (IronPan) [inline components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/README.md:33]: update this as well

File: components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/component.yaml (hunk 1 above)
Comment (ivanmkc) [reply components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/component.yaml]: Done

File: components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/README.md (hunk 2 above)
Comment (ivanmkc) [reply components/google-cloud/google_cloud_pipeline_components/experimental/bigquery/query/to_gcs/README.md]: Done

**Comment by Bobgy** [comment]
/cc @SinaChavoshi @sasha-gitg 
do you want to review?

**Comment by Ark-kun** [comment]
This component looks like a version of Hongye's BigQuery component: https://github.com/kubeflow/pipelines/blob/master/components/gcp/bigquery/query/to_gcs/component.yaml
You should probably keep your version in the same place.

The new components in `components/google-cloud` are based on different code. It would be really confusing to mix them.

**Comment by ivanmkc** [comment]
@IronPan What do you think about @Ark-kun 's suggestion?

**Comment by IronPan** [comment]
We are in the process of merging the two directory and eventually deprecate `components/gcp`, and these PRs would be the first step.
The current state of two directories are confusing.

**Pull request merged**